vsintegration/src/FSharp.Editor/Options/SettingsPersistence.fs

```diff
@@ -18,42 +18,67 @@ type SVsSettingsPersistenceManager = class end
 type SettingsStore(serviceProvider: IServiceProvider) =
 
     let settingsManager = serviceProvider.GetService(typeof<SVsSettingsPersistenceManager>) :?> ISettingsManager
-    // settings quallified type names are used as keys, this should be enough to avoid collisions
-    let storageKey (typ: Type) = typ.Namespace + "." + typ.Name
+
+    let storageKeyVersions (typ: Type) =
+        // "TextEditor" prefix seems to be required for settings changes to be synced between IDE instances
```

Comment:
That's very good to know. Thank you for the comment on this.

Reply:
I should've probably added that it's a lucky guess from looking at Roslyn source 😄. I haven't found it documented anywhere in the open.